### PR TITLE
Disable Tab on Create/Edit Bug

### PR DESCRIPTION
**Pull request by emilyliublair**
Disable Tab on Create/Edit Bug

Issue Number(s): #109 

What does this PR change and why?

This PR tests if user is in edit mode, which is set to true when user is editing/creating a new dog. If they are, variables showFormTab and showLogTab are set to false and they are not displayed.

### Checklist

- [x] Requirements have been implemented
- [x] Acceptance criteria is met
- [x] Database schema [docs](https://www.notion.so/gtbitsofgood/Database-Schema-Docs-a841a38d02db4428a7cdc85dcbb8a35c?pvs=4) have been updated or are not necessary
- [x] Relevant reviewers (EM) have been assigned to this PR

### Critical Changes

N/A

### Related PRs

PR Number(s): N/A

### How To Test

Clicking "Add Dog" or "Edit Dog"

### CODE REVIEW

File: src/pages/dogs/[id]/index.jsx [hunk 1]
```diff
@@ -59,12 +59,19 @@ export default function IndividualDogPage() {
...
+      setShowLogTab(false)
+    } else {
+      setShowFormTab(true)
+      setShowLogTab(true)
+    }
+  }, [isEdit])
 
   // Fetches information about dog if exists and sets correct tabs and filters if needed
   useEffect(() => {
-    setShowLogTab(router.query?.showLogTab);
```
Comment (afazio1) [inline src/pages/dogs/[id]/index.jsx]: This was being used by one of the elements on the Dog table page (`/dogs`). If you click on a tag in the Recent Log Tags section, it is supposed to navigate you to the individual dog page and automatically filter and display the log that the tag is related to. We did this by adding query params `?showLogTab=true&filteredTag=Fleas` to the url. Can you factor this into your current implementation?

showFormTab is being used for a similar purpose: after a user creates a form, the form tab should be active

**Review by afazio1 — CHANGES_REQUESTED** [review]
Awesome work! 🤩 I left one comment about an existing feature that got changed in your implementation. No worries though it was something very niche that I forgot to mention in the ticket!

File: src/pages/dogs/[id]/index.jsx (hunk 1 above)
Comment (emilyliublair) [reply src/pages/dogs/[id]/index.jsx]: done! also, there seems to be a bug in terms of the filtering logic in that it doesn't always work when you click on a tag in the recent log tags section. i'll try to take a look at it this weekend if i get the chance!

**Comment by dograh** [comment]
The design looks good to me. Awesome job!

**Review by afazio1 — APPROVED** [review]

**Review by laurenmcp — APPROVED** [review]
Great job! :D

**Pull request merged**